hera_cal/redcal.py

```diff
@@ -504,7 +506,6 @@ def _solver(self, solver, data, wgts={}, detrend_phs=False, **kwargs):
         Returns:
             solver: instantiated solver with redcal equations and weights
         """
-        # XXX ARP: concerned about detrend_phs.  Why is it necessary?
```

Comment:
But add explanation of why we use detrend_phs.  It's to center the phase bias around 0 to get maximal distance from the +/- pi wrap points.